README.md

```diff
@@ -680,6 +680,23 @@ point that out explicitly and clearly in the associated patches and Cc
   could carry `user.dbus` set to `1`, and Varlink sockets
   `user.varlink` set to `1` and so on.
 
+* Extend `pidfd_open()` to allow opening the thread-group leader based on the
+  PID of an individual thread. Currently we do support:
+
+  (1) `pidfd_open(1234, 0)` on a thread-group lader PID
+  (2] `pidfd_open(1234, PIDFD_THREAD)` on a thread
+
+   Add an option to go from individual thread to thread-group leader.
+
+   **Use-Case:** Allow for a race free way to go from individual thread to
+   thread-group leader pidfd.
+
+* Add a namespace ioctl to translate a PID between PID namespaces.
+
+  **Use-Case:** This makes it possible to e.g., figure out what a given PID in
+  a pid namespace corresponds to in the caller's PID namespace. For example, to
```

Comment:
"pid namespace" vs. "PID namespace". Either way is fine, but please be consistent.